seirmo/tests/test_figures.py

```diff
@@ -0,0 +1,58 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import unittest
+
+import numpy as np
+import pandas as pd
+
+import seirmo as se
+
+
+class IncidenceNumberPlot(unittest.TestCase):
+    """
+    Test the 'IncidenceNumberPlot' class.
+    """
+    def test__init__(self):
+        se.IncidenceNumberPlot()
+
+    def test_add_data(self):
+        pandas_DataFrame = pd.DataFrame({'Time': [0, 1, 2, 3, 4, 5, 6],
+                                         'Incidence Number': [1, 2, 3, 4, 5, 6, 7]}) # noqa
+        time_key = 'Time'
+        inc_key = 'Incidence Number'
+        test_plot = se.IncidenceNumberPlot()
+        test_plot.add_data(pandas_DataFrame, time_key=time_key,
+                           inc_key=inc_key)
+
+        # Test the data in the figure is the same as what we give
```

Comment:
It's better to test the 'x' and 'y' data separately. Because at the moment if either is wrong an error is thrown but you wouldn't know which one. If you check them separately the test would also tell you whether the time or the incidence number are off.